packages/terra-button/src/_themes.scss

```diff
@@ -0,0 +1,34 @@
+@import './mixins';
+
+@include cerner-consumer-theme {
+  @include terra-consumer-theme-button(#64696c, #1c1f21, #383f42);
+
+  /* stylelint-disable selector-class-pattern */
+  .terra-consumer-theme-concept1 {
```

Comment:
Is there any functional direction on when a concept should be used?

Reply:
https://github.com/cerner/terra-core/pull/743#discussion_r134898969